common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -321,6 +321,11 @@ void close() {
     }
   }
 
+  @VisibleForTesting
+  public static File getFileForTest(String[] localDirs, int subDirsPerLocalDir, String filename) {
+    return getFile(localDirs, subDirsPerLocalDir, filename);
+  }
```

Comment:
this is unused now, right?  You can undo this change?

Reply:
yes,your are right,i have fix it.